File: FournilBack/src/main/java/fr/fournil/bakery/model/entities/ProductForm.java
package fr.fournil.bakery.model.entities;



import javax.validation.constraints.Min;
import javax.validation.constraints.NotEmpty;
import javax.validation.constraints.NotNull;




public class ProductForm {
	
	
	 @Min(value = 1)
	    private Integer quantity;
	  
	   @NotNull
	    private  String formatChoosen;
	   @NotNull
	    private String frequencyCommandeProduct;
		public ProductForm() {
			super();
		}
		public ProductForm(@Min(1) Integer quantity, @NotNull String formatChoosen,
				@NotNull String frequencyCommandeProduct) {
			super();
			this.quantity = quantity;
			this.formatChoosen = formatChoosen;
			this.frequencyCommandeProduct = frequencyCommandeProduct;
		}
		public Integer getQuantity() {
			return quantity;
		}
		public void setQuantity(Integer quantity) {
			this.quantity = quantity;
		}
		public String getFormatChoosen() {
			return formatChoosen;
		}
		public void setFormatChoosen(String formatChoosen) {
			this.formatChoosen = formatChoosen;
		}
		public String getFrequencyCommandeProduct() {
			return frequencyCommandeProduct;
		}
		public void setFrequencyCommandeProduct(String frequencyCommandeProduct) {
			this.frequencyCommandeProduct = frequencyCommandeProduct;
		}
		
	    
	    

}

File: FournilFront/src/app/models/product-form.ts
import { Format } from "./format";


export class ProductForm {
private _quantity:Number;
private _formatChoosen:String;
private _frequencyCommandeProduct:String;

	constructor(quantity: Number, formatChoosen: String, frequencyCommandeProduct: String) {
		this._quantity = quantity;
		this._formatChoosen = formatChoosen;
		this._frequencyCommandeProduct = frequencyCommandeProduct;
	}

    /**
     * Getter quantity
     * @return {Number}
     */
	public get quantity(): Number {
		return this._quantity;
	}

    /**
     * Getter formatChoosen
     * @return {String}
     */
	public get formatChoosen(): String {
		return this._formatChoosen;
	}

    /**
     * Getter frequencyCommandeProduct
     * @return {String}
     */
	public get frequencyCommandeProduct(): String {
		return this._frequencyCommandeProduct;
	}

    /**
     * Setter quantity
     * @param {Number} value
     */
	public set quantity(value: Number) {
		this._quantity = value;
	}

    /**
     * Setter formatChoosen
     * @param {String} value
     */
	public set formatChoosen(value: String) {
		this._formatChoosen = value;
	}

    /**
     * Setter frequencyCommandeProduct
     * @param {String} value
     */
	public set frequencyCommandeProduct(value: String) {
		this._frequencyCommandeProduct = value;
	}



	////////////////    
}